CHANGELOG.md

```
@@ -1,52 +1,74 @@
 # [1.3.0](https://github.com/VilnaCRM-Org/website/compare/v1.2.2...v1.3.0) (2025-03-08)
 
+### Bug Fixes
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)